src/components/download/selection.jsx

```diff
@@ -18,43 +18,60 @@ const DownloadSelection = () => {
+                <div className="bg-site-200 rounded-[0.65em] p-2 my-2 w-40">
+                <code>Login: {content.details.login.username} <br /> Password: {content.details.login.password} <br /></code>
+                </div>
             )}
+            <button className="mt-4 hover:scale-105 transition-all bg-rhino-purple rounded-[0.65em] text-white text-xl px-4 py-2 shadow-md mr-2"><a href={content.details.downloadMirror} target="_blank" rel="noopener noreferrer">Download</a></button>
+            <div className="tooltip">
+              <span className="tooltiptext" id={content.title}>Copy to clipboard</span>
+              <button
+                onClick={() => {
+                  navigator.clipboard.writeText(content.details.shasum);
+                  document.getElementById(content.title).innerHTML = "Copied to clipboard!";
+                }}
+                onMouseOut={() => {
+                  document.getElementById(content.title).innerHTML = "Copy to clipboard";
+                }}
+             className="my-4 hover:scale-105 transition-all bg-site-100 p-1 rounded-[0.65em] text-white text-sm px-2 py-1 shadow-md">sha256sum</button>
+            </div>
         </div>
     ));
   };
 
   return (
-    <div className="md:w-[65%] w-full mx-auto md:p-0 p-4">
-      <label htmlFor="platform" className="block text-sm font-medium text-white">Select a platform:</label>
-      <select
-        id="platform"
-        value={selectedPlatform}
-        onChange={(e) => setSelectedPlatform(e.target.value)}
-        className="block w-full mt-2 p-2 bg-indigo-950 text-white text-xl border-2 border-rhino-purple rounded-md shadow-sm focus:ring-indigo-500 focus:border-indigo-500"
-      >
-        <option value="generic">Generic (x86_64/ARM64)</option>
-        <option value="pine">Pine64 (ARM64)</option>
-        <option value="rpi">Raspberry Pi (ARM64)</option>
-      </select>
+    <div className="md:w-[85%] w-full mx-auto md:p-0 p-4">
+      <h1 className="text-off-white text-center pt-8 text-3xl drop-shadow-header-drop">
+        Download Rhino Linux
+      </h1>
+      <h2 className="text-off-white text-center text-2xl font-light mt-2 drop-shadow-header-drop">
+        Version <b><b>2024.2</b></b>
+      </h2>
+      <div className="mx-2">
+        <select
+          id="platform"
+          value={selectedPlatform}
+          onChange={(e) => setSelectedPlatform(e.target.value)}
+          className="p-2 px-4 bg-site-300 w-[100%] text-off-white select-fix md:text-2xl font-light rounded-[0.5em] mt-4 appearance-none"
+        >
+          <option value="generic">Generic (x86_64/ARM64)</option>
+          <option value="pine">Pine64 (ARM64)</option>
+          <option value="rpi">Raspberry Pi (ARM64)</option>
+        </select>
+      </div>
 
-      <div className="content-display mt-6">
+      <div className="content-display mt-6 flex-grid">
```

Comment:
remove mt-6, it adds too much vertical padding between the selection box and the downloads. 
![Padding issues](https://github.com/user-attachments/assets/4677a54c-5de8-498f-8694-9a8532cf145e)